pkg/node/utils.go

```diff
@@ -162,8 +162,12 @@ func FindDrives(ctx context.Context, nodeID string) ([]*direct_csi.DirectCSIDriv
 	for _, v := range drives {
 		if v.Name != v.RootPartition {
 			root := drives[v.RootPartition]
-			v.ModelNumber = fmt.Sprintf("%s-part%d", root.ModelNumber, v.PartitionNum)
-			v.SerialNumber = fmt.Sprintf("%s-part%d", root.SerialNumber, v.PartitionNum)
+			if root.ModelNumber != "" {
+				v.ModelNumber = fmt.Sprintf("%s-part%d", root.ModelNumber, v.PartitionNum)
```

Comment:
if empty then the serial number and model number are "-part". This change fixes it